src/backend/storeManagers/sideload/games.ts

```diff
@@ -71,7 +72,8 @@ export async function launch(
   /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
   launchArguments?: LaunchOption
 ): Promise<boolean> {
-  return launchGame(appName, getGameInfo(appName), 'sideload')
+  // wait for the game to launch and finish running
+  return Promise.resolve(launchGame(appName, getGameInfo(appName), 'sideload'))
```

Comment:
This does not do anything, calling `Promise.resolve` with a promise just returns that promise